From 87e24b26adead260b45e65740c7b6b9343591a7c Mon Sep 17 00:00:00 2001
From: Vishnu Saini <vishsain@qti.qualcomm.com>
Date: Thu, 20 Nov 2025 17:58:47 +0530
Subject: [PATCH 4/4] FROMLIST: arm64: dts: qcom: add lt8713sx bridge for iq8

Add device tree node for the lontium lt8713sx bridge support.

Change-Id: I064217f76933a3405654b0905e590d9319fb3c81
Link: https://lore.kernel.org/r/20251120-lt8713sx-bridge-linux-for-next-v1-0-2246fc5fb490@qti.qualcomm.com
Signed-off-by: Prahlad Valluru <vvalluru@qti.qualcomm.com>
Signed-off-by: Vishnu Saini <vishsain@qti.qualcomm.com>
Upstream-Status: Pending

---
 .../boot/dts/qcom/qcs8275-iq-8275-evk-pro-sku.dts  | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/qcs8275-iq-8275-evk-pro-sku.dts b/arch/arm64/boot/dts/qcom/qcs8275-iq-8275-evk-pro-sku.dts
index ac0c6d3f5fd3..bbbd4627c621 100644
--- a/arch/arm64/boot/dts/qcom/qcs8275-iq-8275-evk-pro-sku.dts
+++ b/arch/arm64/boot/dts/qcom/qcs8275-iq-8275-evk-pro-sku.dts
@@ -317,6 +317,20 @@ &gpi_dma1 {
 	status = "okay";
 };
 
+&i2c0 {
+	pinctrl-0 = <&qup_i2c0_default>;
+	pinctrl-names = "default";
+
+	status = "okay";
+
+	lt8713sx: lt8713sx@4f {
+		/*Display bridge chip, DP1.4/HDMI2.0/DP++ hub*/
+		compatible = "lontium,lt8713sx";
+		reg = <0x4f>;
+		reset-gpios = <&expander5 6 GPIO_ACTIVE_HIGH>;
+	};
+};
+
 &i2c1 {
 	pinctrl-0 = <&qup_i2c1_default>;
 	pinctrl-names = "default";
-- 
2.25.1

